var/spack/repos/builtin/packages/util-linux/package.py

```diff
@@ -37,7 +41,14 @@ def configure_args(self):
         config_args = [
             '--disable-use-tty-group',
             '--disable-makeinstall-chown',
-            '--without-systemd'
+            '--without-systemd',
         ]
+        if "+bash" in self.spec:
+            config_args.extend(
+                ['--enable-bash-completion',
+                 '--bash-completion-dir=' + self.spec['bash'].prefix])
```

Comment:
Does this get put into a subdirectory like share or does it go directly in the prefix?

Reply:
Yep, good catch.

Looking at where it tries to install in /usr I've set the correct directory.

Theoretically it should be getting the correct path from pkg-config, so we might not even need this flag. Unfortunately I can't check against a fresh install of bash because `spack install bash` is broken for me, but